.github/workflows/active-record-multi-tenant-tests.yml

```diff
@@ -49,26 +49,26 @@ jobs:
           - '3.2'
           - '3.3'
         appraisal:
-          - rails-6.0
-          - rails-6.1
-          - rails-7.0
-          - rails-7.1
-          - rails-7.2
-          - rails-8.0
-          - active-record-6.0
-          - active-record-6.1
-          - active-record-7.0
-          - active-record-7.1
-          - active-record-7.2
-          - active-record-8.0
+          - rails_6.0
+          - rails_6.1
+          - rails_7.0
+          - rails_7.1
+          - rails_7.2
+          - rails_8.0
+          - active_record_6.0
+          - active_record_6.1
+          - active_record_7.0
+          - active_record_7.1
+          - active_record_7.2
+          - active_record_8.0
```

Comment:
I think `rails` and `active_record` will mean almost the same thing, so only `rails` is fine.